core/src/main/java/org/apache/iceberg/BaseFilesTable.java

```diff
@@ -115,7 +115,7 @@ protected BaseAllFilesTableScan(TableOperations ops, Table table, Schema schema,
     }
 
     /**
-     * Returns an iterable of manifest files to explore for this all files metadata table scan
+     * Returns list of {@link ManifestFile} files to explore for this file's metadata table scan
```

Comment:
This isn't correct. Can you revert these changes?

Reply:
reverted